apps/jobs/metrics.py

```diff
@@ -0,0 +1,6 @@
+from prometheus_client import Counter
+
+submissions_loaded_in_queue = Counter(
```

Comment:
@burnerlee this is not correct. We need a wrapper on `Counter` class that will have an `inc` method. This wrapper will create a prometheus counter object with tags and labels, etc. All we have to do in an API is create one object of the Counter wrapper and use it to make `increment` calls for tags